tools/vscode/src/inspect/props.ts

```diff
@@ -118,11 +118,19 @@ export function inspectViewPath(): AbsolutePath | null {
           version: string;
           path: string;
         };
-        const viewPath = toAbsolutePath(version.path)
+        let viewPath = toAbsolutePath(version.path)
           .child("_view")
-          .child("www");
+          .child("www")
+          .child("dist");
         if (viewPath) {
```

Comment:
I think we might need an existence check here? (as it looks like `toAbsolutePath` will always return a valid object). That `if (viewPath)` is probably residual that needs to be removed. @dragonstyle  ?

Reply:
Yes, I agree (and that check is effectively doing nothing and just always setting the value in the cache). I can fix this when I am looking at any other possible vscode related changes (e.g. the html for the viewer) and add a commit to address this.